solarwinds_apm/apm_config.py

```diff
@@ -134,17 +135,29 @@ def __init__(
             self.__config["service_key"],
             self.service_name,
         )
+        self.update_log_type()
+        # update logging level of Python logging logger
+        apm_logging.set_sw_log_level(self.__config["debug_level"])
```

Comment:
Moved this up here instead of in `_set_config_value` for clarity and so only called once.